internal/locald/sandboxmanager/sandbox_monitor.go

```diff
@@ -35,28 +40,66 @@ func newSBMonitor(rk string, clapiClient clapiclient.Client, rtClient revtun.Cli
 		revtunClient: rtClient,
 		delFn:        delFn,
 		log:          log,
-		done:         make(chan struct{}),
+		doneCh:       make(chan struct{}),
+		reconcileCh:  make(chan struct{}, 1),
 		locals:       make(map[string]*models.Local),
 		revtuns:      make(map[string]*rt),
 	}
 	go res.monitor()
 	return res
 }
 
+func (sbm *sbMonitor) getStatus() *clapi.WatchSandboxStatus {
+	sbm.Lock()
+	defer sbm.Unlock()
+	return sbm.status
+}
+
+func (sbm *sbMonitor) stop() {
+	select {
+	case sbm.doneCh <- struct{}{}:
```

Comment:
I understand it is not needed in this particular case, but I'd still prefer to use `close()`, as it is more idiomatic and easier to evolve the code -- for example we could have to determine if it is closed after some code changes.